packages/grafana-ui/src/components/Card/CardContainer.tsx

```diff
@@ -39,6 +39,8 @@ export interface CardContainerProps extends HTMLAttributes<HTMLOrSVGElement>, Ca
   disableEvents?: boolean;
   /** No style change on hover */
   disableHover?: boolean;
+  /** Makes the card selectable, set to "true" to apply selected styles */
```

Comment:
**Praise:** although I usually dislike comments, these ones that make the props easier to understand I like a lot - in case they are correct they can spare an extra look at the stories / tests / docs.